azuread/table_azuread_application.go

```diff
@@ -77,10 +77,7 @@ func listAdApplications(ctx context.Context, d *plugin.QueryData, _ *plugin.Hydr
 	client := msgraph.NewApplicationsClient(session.TenantID)
 	client.BaseClient.Authorizer = session.Authorizer
 
-	// As per our test result we have set the max limit to 999
```

Comment:
@ParthaI Was this change tested for this table, or for any other tables, where there were results over 999 and we got back all of them correctly? How does the hamilton SDK we're using handle large data sets and pagination?